styleguide/source/_patterns/01-atoms/table.twig

```diff
@@ -0,0 +1,42 @@
+<table>
+  <caption>How I chose to spend my money</caption>
+  <thead>
+  <tr>
+    <th data-title="Purchase">Purchase</th>
+    <th data-title="Location">Location</th>
```

Comment:
Can we use loops for these? I think this a perfect opportunity to makes this dryer